apps/frontend/src/libs/components/header/styles.module.css

```
@@ -17,14 +22,8 @@
 	gap: 8px;
 }
 
-@media (width <= 991px) {
-	.header {
-		padding-inline: 32px;
-	}
-}
-
 @media (width <= 768px) {
-	.header {
-		padding-inline: 16px;
+	.header-container {
+		padding: 10px 16px;
```

should be divisible by 4, so either use 8 or 12
```suggestion
		padding: 8px 16px;
```